dist/wavesurfer.js

```diff
@@ -1192,14 +1192,15 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
      *  @param  {String}        url         path to media file
      *  @param  {HTMLElement}   container   HTML element
      *  @param  {Array}         peaks       array of peak data
+	 *  @param  {String}        preload     HTML 5 preload attribute value
```

Comment:
@katspaugh Sorry for the inconvenience. Indentations are not displayed and created equally between editors. --'